api-inference-community/docker_images/common/app/pipelines/text_classification.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any, Dict
+
+from app.pipelines import Pipeline
+
+
+class TextClassificationPipeline(Pipeline):
+    def __init__(
+        self,
+        model_id: str,
+    ):
+        # IMPLEMENT_THIS
+        # Preload all the elements you are going to need at inference.
+        # For instance your model, processors, tokenizer that might be needed.
+        # This function is only called once, so do all the heavy processing I/O here
+        raise NotImplementedError(
+            "Please implement TextClassificationPipeline __init__ function"
+        )
+
+    def __call__(self, inputs: str) -> Dict[str, Any]:
+        """
+        Args:
+            inputs (:obj:`str`):
+                a string containing some text
+        Return:
+            A :obj:`dict`:. The object returned should be like {"label": 0.9939950108528137} containing :
```

Comment:
In the case of multiple labels it would actually be a list of dicts. 

Something like this:

```
[
            {"label": "NEGATIVE", "score": 0.0001261125144083053},
            {"label": "POSITIVE", "score": 0.9998738765716553},
]
```

I think both the docstring and the typing should be updated to reflect this.

Reply:
Thank you! :D I will get to these soon